tests/buyers/views/test_buyers.py

```diff
@@ -2255,7 +2244,7 @@ def test_404_if_framework_is_not_live_or_expired(self):
     def test_get_responses(self):
         brief = mock.Mock()
 
-        with po(buyers, 'get_sorted_responses_for_brief') as m:
+        with mock.patch.object(buyers, 'get_sorted_responses_for_brief', autospec=True) as m:
```

Comment:
So that `po` method was defined as a way to limit line length.
Problems I had with it were:
    It was used in some places and not in others (ie. file wide, codebase wide, multiple codebase wide)
    It's a bit of an arcane construct. Makes our code less accessible
    It became unnecessary (re line length) with a bit of shifting around

I'll remove it globally in this PR, in the updates I'm making today, unless you feel strongly about keeping it